### PR TITLE
Group branch

### PR DESCRIPTION
**Pull request by Masouma96**
Create a database based on the diagram

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @Masouma864 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that required a commit massage from your partner but only you commit to this PR.

- [ ] Please invite your partner to contribute to this PR.

- [ ] Please check your README file and update all broken links and update details for the different headers.
- [ ] Please add more foreign key index for:
`CREATE INDEX ON medical_histories (patient_id);`
`CREATE INDEX ON invoices (medical_history_id);`
`CREATE INDEX ON invoice_items (invoice_id);`
`CREATE INDEX ON invoice_items (treatment_id);`


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+CREATE TABLE invoice_items (
+  id BIGSERIAL PRIMARY KEY,
+  unit_price DECIMAL NOT NULL,
+  quantity INT NOT NULL,
+  total_price DECIMAL NOT NULL,
+  invoice_id INT REFERENCES invoices(id) NOT NULL,
+  treatement_id INT REFERENCES treatments(id)
+);
+
```
Comment (BenMukebo) [inline schema_based_on_diagram.sql:36]: - [ ] Great job so far 👏🏻 however as requested by the previous reviewer, let's make sure all FOREIGN KEYS have indexes created, Eg:

``` ruby
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
    ...
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
- The new tables are created in `schema_based_on_diagram.sql` file. 👍🏻 
- All tables from the diagram are created 🥇 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
### Hi @Team, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 

- All FOREIGN KEYS have indexes created🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**